test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using Microsoft.ML.Functional.Tests.Datasets;
+using Microsoft.ML.RunTests;
+using Microsoft.ML.TestFramework;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Trainers.FastTree;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Functional.Tests
+{
+    public class ModelFiles : BaseTestClass
+    {
+        public ModelFiles(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        /// <summary>
+        /// Model Files: The (minimum) nuget version can be found in the model file.
+        /// </summary>
+        [Fact]
+        public void DetermineNugetVersionFromModel()
+        {
+            var modelFile = GetDataPath(@"backcompat" + Path.DirectorySeparatorChar + @"keep-model.zip");
+            var versionFileName = @"TrainingInfo\Version.txt"; // Can't find this cross plat.
```

Comment:
>\ [](start = 48, length = 1)

Path.DirectorySeparatorChar

Reply:
Strangely enough, since the zip archive was made on Windows, we need to specify a '\' here. I've updated this cryptic comment to explain it better.

---
In reply to: [265712860](https://github.com/dotnet/machinelearning/pull/2899#discussion_r265712860) [](ancestors = 265712860)